<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanFlightDialog</class>
 <widget class="QDialog" name="PlanFlightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Routes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/qutescoop.png</normaloff>:/icons/qutescoop.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="gbEssentials">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Route essentials</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lblDep">
          <property name="text">
           <string>de&amp;p</string>
          </property>
          <property name="buddy">
           <cstring>edDep</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edDep">
          <property name="maxLength">
           <number>4</number>
          </property>
          <property name="placeholderText">
           <string>ICAO</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="bDepDetails">
          <property name="toolTip">
           <string>Open airport dialog (EDDS)</string>
          </property>
          <property name="text">
           <string>EDDS 3/1</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="edDest">
          <property name="maxLength">
           <number>4</number>
          </property>
          <property name="placeholderText">
           <string>ICAO</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QToolButton" name="bDestDetails">
          <property name="toolTip">
           <string>Open airport dialog (EDDS)</string>
          </property>
          <property name="text">
           <string>EDDS 3/1</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblDest">
          <property name="text">
           <string>des&amp;t</string>
          </property>
          <property name="buddy">
           <cstring>edDest</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Select providers</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,5,0">
        <item row="1" column="4">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="edGenerated">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>20</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="lblGeneratedStatus">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="pixGenerated">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="Resources.qrc">:/icons/qutescoop.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>cbGenerated</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="pixVroute">
          <property name="pixmap">
           <pixmap resource="Resources.qrc">:/routeproviders/images/vroute.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="5" rowspan="2">
         <widget class="QLabel" name="lblVrouteStatus">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="6">
         <widget class="Line" name="line_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="cbGenerated">
          <property name="text">
           <string>user</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>your route</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="4" rowspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
          <item>
           <widget class="QLineEdit" name="edCycle">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>44</height>
             </size>
            </property>
            <property name="maxLength">
             <number>4</number>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>yymm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblVroute">
            <property name="text">
             <string>&lt;a href=&quot;http://www.vroute.net&quot;&gt;www.vroute.net&lt;/a&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="5" column="2">
         <widget class="QCheckBox" name="cbVroute">
          <property name="text">
           <string>&amp;vroute</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="lblAirac">
          <property name="toolTip">
           <string>Based on the given Airac (nav database) cycle. Leave blank for standard.</string>
          </property>
          <property name="text">
           <string>Airac cycle</string>
          </property>
          <property name="buddy">
           <cstring>edCycle</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="6">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="5">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="0" colspan="6">
         <widget class="QPushButton" name="buttonRequest">
          <property name="text">
           <string>r&amp;equest</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbResults">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Results [LSZH-EPWA]: 0</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeView" name="treeRoutes">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pbCopyToClipboard">
           <property name="text">
            <string>&amp;copy to clipboard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbVatsimPrefile">
           <property name="toolTip">
            <string>Launches prefile page in your browser.</string>
           </property>
           <property name="text">
            <string>&amp;VATSIM prefile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
     <item>
      <widget class="QCheckBox" name="cbPlot">
       <property name="toolTip">
        <string>Needs a navigation database (see Preferences/Special)</string>
       </property>
       <property name="text">
        <string>&amp;plot route</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="linePlotStatus">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPlotStatus">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>waypoints (calculated):...</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edDep</tabstop>
  <tabstop>edDest</tabstop>
  <tabstop>cbGenerated</tabstop>
  <tabstop>edGenerated</tabstop>
  <tabstop>edCycle</tabstop>
  <tabstop>treeRoutes</tabstop>
  <tabstop>pbCopyToClipboard</tabstop>
  <tabstop>pbVatsimPrefile</tabstop>
  <tabstop>cbPlot</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbGenerated</sender>
   <signal>toggled(bool)</signal>
   <receiver>edGenerated</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbVroute</sender>
   <signal>toggled(bool)</signal>
   <receiver>pixVroute</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbVroute</sender>
   <signal>toggled(bool)</signal>
   <receiver>edCycle</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGenerated</sender>
   <signal>toggled(bool)</signal>
   <receiver>pixGenerated</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>PlanFlightDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbVroute</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblAirac</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGenerated</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
